tx.go

```diff
@@ -457,10 +457,20 @@ func (tx *Tx) buildListIdx(bucket string, entry *Entry) {
 	if _, ok := tx.db.ListIdx[bucket]; !ok {
 		tx.db.ListIdx[bucket] = list.New()
 	}
-
 	key, value := entry.Key, entry.Value
-
+	if IsExpired(entry.Meta.TTL, entry.Meta.Timestamp) {
+		return
+	}
 	switch entry.Meta.Flag {
+	case DataExpireListFlag:
+		t, _ := strconv2.StrToInt64(string(value))
+		ttl := uint32(t)
+		list, _ := tx.db.ListIdx[bucket].LRange(string(key), 0, -1)
+		tx.db.ListIdx[bucket].TTL[string(key)] = ttl
+		tx.db.ListIdx[bucket].TimeStamp[string(key)] = entry.Meta.Timestamp
+		for i, element := range list {
```

Comment:
If there are many items in the list, the efficiency is very low, so don’t implement it in this way

Reply:
we also can delete this logic， it is not need

Reply:
I will reslove it soon